### PR TITLE
Use Promise.allSettled for shows query and count query

### PR DESCRIPTION
**Pull request by themisterholliday**
Covering: https://github.com/syntaxfm/website/issues/1442

Simply a quick promise all settled and the required checks for the results.

### CODE REVIEW

File: src/routes/(site)/shows/+page.server.ts [hunk 1]
```diff
@@ -43,9 +46,12 @@ export const load: PageServerLoad = async function ({ locals, url, setHeaders })
 		throw redirect(302, `/shows?${params_string}`);
 	}
 
+	const { value: shows } = shows_result;
+	const total_show_count = count_result.status === 'fulfilled' ? count_result.value : 42;
```
Comment (wesbos) [inline src/routes/(site)/shows/+page.server.ts:50]: What is the 42?

Comment (themisterholliday) [reply src/routes/(site)/shows/+page.server.ts:50]: It is only a default value if the count promise is rejected. Not graceful, but I didn't want to make significant changes.

Comment (wesbos) [reply src/routes/(site)/shows/+page.server.ts:50]: Gotcha, could we just change this all to promise.all? If the DB query fails, we are in bigger trouble and should just let the error page show.

Comment (themisterholliday) [reply src/routes/(site)/shows/+page.server.ts]: Of course! Also done 👍

**Comment by wesbos** [comment]
thanks  a ton!

**Pull request merged**